pages/verification/index.tsx

```diff
@@ -69,7 +69,7 @@ export default function Verification(props: PageProps) {
                                     anon_user_info_color_name: data.anon_user_info_color_name,
                                     anon_user_info_emoji_code: data.anon_user_info_emoji_code,
                                     anon_user_info_emoji_name: data.anon_user_info_emoji_name,
-                                    members: [member],
+                                    member: member,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The property name has been changed from `members` to `member`. This could potentially cause issues if the backend service expects an array of members instead of a single member. Please ensure that this change aligns with the expected data structure on the server side.

```diff
- members: [member],
+ member: member,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->